index.html

```diff
@@ -36,29 +36,130 @@
+                </div>
 
-            <!-- real people should not fill this in and expect good things - do not remove this or risk form bot signups-->
-            <div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_84b372b9598d22040af9583e0_0def27f7e4" tabindex="-1" value=""></div>
+                <!-- real people should not fill this in and expect good things - do not remove this or risk form bot signups-->
+                <div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_84b372b9598d22040af9583e0_0def27f7e4" tabindex="-1" value=""></div>
 
-            <button type="submit" name="subscribe" id="mc-embedded-subscribe" class="mdl-button mdl-button--raised mdl-button--primary">
-                Subscribe
-            </button>
+                <button type="submit" name="subscribe" id="mc-embedded-subscribe" class="mdl-button mdl-button--raised mdl-button--primary">
+                    Subscribe
+                </button>
+                </div>
+            </form>
+        </div>
+    
+    </div>
+
+    <div id="Foodprint" class="tabcontent">
+     <!--    <h3>Foodprint</h3>
+    
+                width="175"
+                height="296"
+    
+    -->
+        <p> 
+            Coming soon... A recipe and diet app with ecology and health at its heart.
+        </p>
+        <p>    
+            Find the footprint behind your plate by searching the nutrition and ecological impacts 
+            of a growing list food items.
+        </p>
+        <p>    
+            The nutritional data is sourced from the USDA.
+            The ecological data is estimated by us from recent scientific studies.
+        </p>
+        <p>    
+            Our alpha app is now online! It is under development and still growing every day. 
+    </div> 
+    
+    <div id="News" class="tabcontent">
+        <p style="text-align:left;"> </p>
+    </div>
+        
+    <div id="Team" class="tabcontent">
+        <div class="row">
+            <div class="column">
+                <img src="evert.png" alt="Evert">  <br>
+                <h5> Evert - CTO </h5>
+                Computer scientist <br>
+                Likes: Backends <br>
+                Dislikes: Bikeshedding
+            </div>
+            <div class="column">
+                <img src="robert.png"
+                
+                style="width: 175px; height: 296px; object-fit: cover;"
+                
+                alt="Robert"> <br>
+                <h5> Robert - CEO </h5>
+                Mathematical engineer<br>
+                Likes: Data<br>
+                Dislikes: Balance fallacy
             </div>
-        </form>
+        </div>
     </div>
 
+      
+    <div id="Contact" class="tabcontent">
+    <p>
+        If you find any bugs or have any sprouts (suggestions) for us, please contact us at contact@quanteco.org.         
```

Comment:
Make it a mailto `<a href="mailto:contact@quanteco.org">contact@quanteco.org</a>`